hub/tests/common.py

```diff
@@ -44,3 +46,39 @@ def current_test_name() -> str:
     test_name = full_name.split("::")[1]
     output = os.path.join(test_file, test_name)
     return output
+
+
+def get_random_array(shape: Tuple[int], dtype: str) -> np.ndarray:
+    dtype = dtype.lower()
+
+    if "int" in dtype:
+        low = np.iinfo(dtype).min
+        high = np.iinfo(dtype).max
+        return np.random.randint(low=low, high=high, size=shape, dtype=dtype)
+
+    if "float" in dtype:
+        # get float16 because np.random.uniform doesn't support the `dtype` argument.
```

Comment:
Still doesn't explain why float16 and not `np.finfo(dtype)`

Reply:
true, the reason is because `np.random.uniform` only supports `float16` and doesn't have a way to overwrite. so the max/min values are `float16` but the return value is casted to `dtype`. will update this comment